external-api/src/v1/controllers/email.controller.ts

```diff
@@ -57,6 +71,14 @@ export const emailNotification = async (req: Request, res: Response) => {
 
 export const sendEmail = async (templateId: string, sendToEmailAddress: string, emailVariables: object) => {
   try {
+    const notifyKey: string = process.env.GOV_NOTIFY_API_KEY || '';
+    const mockNotifyKey: string = process.env.MOCK_E2E_GOV_NOTIFY_API_KEY || '';
+
+    if (notifyKey === mockNotifyKey) {
```

Comment:
Might be prudent to make a central function in `libs/common` `isGovNotifyMocked()` to keep the code DRY and logic abstracted.